di_website/common/templatetags/form_utils.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+register = template.Library()
+
+
+@register.filter
+def placeholder(field, page):
+    try:
+        return page.get_placeholder_for_field(field)
+    except Exception:
```

Comment:
Is there a specific Exception we can add here? Good to be specific so we don't catch unintentional errors